docs/modules/ROOT/pages/user-code-deployment.adoc

```diff
@@ -0,0 +1,219 @@
+= User Code Deployment
+:description: Hazelcast allows you to deploy your own custom and domain classes to cluster members. From Hazelcast Platform Operator, you can deploy your application code from external buckets and ConfigMap objects in Kubernetes. ]
+
+{description}
+
+== Configuring User Code Deployment
+
+Below are the configuration options for the User Code Deployment feature.
+
+[cols="20%m,80%a"]
+|===
+|Field|Description
+
+|`clientEnabled`
+|When set to true, this option enables custom code deployment from Hazelcast clients.
+
+|`bucketConfig`
```

Comment:
Just need to add a line before the two bullet points in the table because they are not showing correctly on the page. Take a look on the Netlify preview: https://deploy-preview-44--pedantic-goldberg-f76ec1.netlify.app/operator/latest-snapshot/user-code-deployment.